### PR TITLE
initial commit for docker dev env set up with readme

### PR DESCRIPTION
**Pull request by smalho01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,275 @@
...
+
+    `<drlsroot>` will be the base directory into which all the other components will be installed. For example, CRD will be cloned to `<drlsroot>/crd`.
+
+    Note: If you are using a different project structure from the above description, you will need to change the corresponding repo paths in docker-compose-prior-auth-dev.yml, docker-sync.yml, and docker-compose.yml
+
+2. Now clone the DRLS component repositories from Github:
+    ```bash
+    cd <drlsroot>
+    https://github.com/HL7-DaVinci/DRLS-Docker.git DRLS-Docker 
+    git clone https://github.com/HL7-DaVinci/CRD.git CRD
```
Comment (smalho01) [inline README.md:136]: using mcode CRD repo for REMS should still work

File: README.md [hunk 2]
```diff
@@ -1,2 +1,275 @@
...
+    git clone https://github.com/HL7-DaVinci/CDS-Library CDS-Library
+    git clone https://github.com/HL7-DaVinci/test-ehr test-ehr
+    git clone https://github.com/HL7-DaVinci/crd-request-generator crd-request-generator
+    git clone https://github.com/HL7-DaVinci/dtr dtr
```
Comment (smalho01) [inline README.md:140]: sing mcode CRD repo for REMS should still work

File: README.md [hunk 3]
```diff
@@ -1,2 +1,275 @@
...
+        rbenv init
+    ```
+3. Close Terminal so changes take affect
+4. Test rbenv is installed correctly 
+  ```bash
+        curl -fsSL https://github.com/rbenv/rbenv-installer/raw/main/bin/rbenv-doctor | bash    
+    ```
+5. Install Ruby 
+  ```bash
+        ruby install 3.0.2
```
[the remark this answers is not in the document]
Comment (smalho01) [reply README.md:88]: yes I did mean rbenv install, will update

**Comment by danlee1025** [comment]
The README mentions installing ruby `3.0.2` to install `docker-sync 0.7.0`. When I was on ruby 3 it would not let me install version `0.7.0` of `docker-sync`. I was able to finally install the `docker-sync 0.7.0` when I went to ruby `2.7.2`. I'm not sure if this is just something weird with my environment but wanted to post this in case others are running into the same issue.

**Pull request closed without merging**